dotCMS/src/main/java/com/dotcms/rest/api/v1/experiment/AnalyticEventTypeHandler.java

```diff
@@ -0,0 +1,9 @@
+package com.dotcms.rest.api.v1.experiment;
+
+import java.util.Map;
+
+public interface AnalyticEventTypeHandler {
```

Comment:
Rename to "AnalyticsEventTypeHandler"

Reply:
it could be a FunctionalInterface, also add doc, it is not so much self explained

Reply:
I think this should be called, EventHandler, b/c the function seems to be handle the event information, not the type, the type is just a pivot to classified the objects